### ISSUE TITLE
Signer key pair in derive example needed?

### ISSUE THREAD

**Issue by F-Node-Karlsruhe** [digitalbazaar/ecdsa-sd-2023-cryptosuite#6]
Is the signer keyPair in the derive example (https://github.com/digitalbazaar/ecdsa-sd-2023-cryptosuite/blob/main/README.md?plain=1#L164) really needed? From the spec and the implementation (https://github.com/digitalbazaar/ecdsa-sd-2023-cryptosuite/blob/25ddc8128a73758a3e2d1eaac60e1fc0be928bae/lib/disclose.js#L135) this should not be the case afaiu. or is the public key of the issuer needed for the disclosure? If so it could be found within the credential itself in order to seperate holder and issuer conceptually as the issuer is not needed for partially disclosing credentials.

Maybe i'm wrong but i found this a bit confusing in the example.

**Comment by dlongley** [comment]
@F-Node-Karlsruhe,

Yes, that was a typo in the README, thanks for noticing it! It's been fixed in 62c76663f81cb6f836efd5f28109ae3ede113e5d.
